msteams-platform/sbs-gs-blazorapp.yml

```diff
@@ -0,0 +1,387 @@
+### YamlMime:Tutorial
+title: Build your first app using Blazor
+metadata:
+  title: Hello World with Blazor
+  description: In this tutorial, you'll learn to build Hello World app with Blazor.
+  audience: Developer
+  level: Beginner
+  ms.date: 04/03/2022
+  ms.topic: interactive-tutorial
+  nextTutorialHref: ~/get-started/get-started-overview.md
+  nextTutorialTitle: Back to Get started overview
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+
+    Blazor lets you build interactive web UIs using C#, instead of JavaScript. 
+    
+    You'll create a tab app with Blazor and the latest version of Visual Studio.
+
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/blazor/your-blazor-helloworld-apps.png" alt-text="Diagram showing this app has two features, tab and Bot." border="false":::
+
+    > [!NOTE]
+    > Currently, Teams Toolkit offers templates for only a tab app with Blazor. Teams Toolkit doesn't support bot and message extension capability.
+
+- title: Prerequisites
+  durationInMinutes: 1
+  content: |
+
+    Here's a list of tools you'll need for building and deploying your app.
+
+    | &nbsp;    | Install   | For using... |
+    | --- | --- | --- |
+    | **Required** | &nbsp; | &nbsp; |
+    | &nbsp; | [Latest version of Visual Studio](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=enterprise&ch=pre&rel=17), **ASP.NET and web development**, or **.NET Core cross-platform development** workload | .NET. Choose Visual Studio Enterprise 2022 Preview (version 17.2.0). |
+    | &nbsp; | - Visual Studio Code Extensions | •  [Azure Functions Core Tools](/azure/azure-functions/functions-run-local): Backend components locally during a local debug run. It includes the authentication helpers required when running your services in Azure. It's installed within the project directory (using the npm `devDependencies`). </br> •  [.NET SDK](/dotnet/core/install/): Customized bindings for local debugging and Azure Functions app deployments. If you have not installed the .NET 3.1 (or later) SDK globally, the portable version can be installed. |
```

Comment:
Do we have dependency on VSCode?

Reply:
right. we do not have dependency on VS Code. need to remove that